changelog.md

```diff
@@ -2,6 +2,13 @@
 
 //  Add your own contribution below
 
+* Add `danger.utils` DSL, which includes `danger.utils.href()` and `danger.utils.sentence()` - macklinu
+
+```js
+danger.utils.href("http://danger.systems", "Danger") // <a href="http://danger.systems">Danger</a>
+danger.utils.sentence(["A", "B", "C"]) // "A, B and C"
```

Comment:
If you have suggestions for a better changelog entry (both the entry and the examples), I'm all ears. 👂 👍